package/MDAnalysis/lib/formats/libdcd.pyx

```diff
@@ -609,7 +566,6 @@ cdef class DCDFile:
         self.current_frame += 1
         return DCDFrame(xyz, unitcell)
 
-
     def readframes(self, start=None, stop=None, step=None, order='fac', indices=None):
```

Comment:
maybe deprecate this method for 3.0?  Or if we want to keep it, seems like we could handle this sort of behaviour in a parent class if we really thought it was useful

Reply:
Issue raised #3930